dashboard/lib/projects_list.rb

```diff
@@ -60,18 +60,99 @@ def fetch_section_projects(section)
     #   when requesting all project types. Optional.
     # @return [Hash<Array<Hash>>] A hash of lists of published projects.
-    def fetch_published_projects(project_group, limit:, published_before:)
+    def fetch_published_projects(project_group, limit:, published_before:, append_featured:)
       unless limit && limit.to_i >= 1 && limit.to_i <= MAX_LIMIT
         raise ArgumentError, "limit must be between 1 and #{MAX_LIMIT}"
       end
       if project_group == 'all'
         raise ArgumentError, 'Cannot specify published_before when requesting all project types' if published_before
-        return fetch_published_project_types(PUBLISHED_PROJECT_TYPE_GROUPS.keys, limit: limit)
+        if append_featured
+          return include_featured(limit: limit)
+        else
+          return fetch_published_project_types(PUBLISHED_PROJECT_TYPE_GROUPS.keys, limit: limit)
+        end
       end
       raise ArgumentError, "invalid project type: #{project_group}" unless PUBLISHED_PROJECT_TYPE_GROUPS.keys.include?(project_group.to_sym)
       fetch_published_project_types([project_group.to_s], limit: limit, published_before: published_before)
     end
 
+    def include_featured(limit:)
+      published = fetch_published_project_types(PUBLISHED_PROJECT_TYPE_GROUPS.keys, limit: limit)
+      featured = fetch_featured_published_projects
+      featured_channel_ids = []
+      PUBLISHED_PROJECT_TYPE_GROUPS.keys.each do |project_type|
+        featured[project_type].each do |project|
+          featured_channel_ids << project["channel"]
+        end
+      end
+      PUBLISHED_PROJECT_TYPE_GROUPS.keys.each do |project_type|
+        published[project_type].each do |project|
+          unless featured_channel_ids.include?(project["channel"])
+            featured[project_type].push(project)
+          end
+        end
+      end
+      return featured
+    end
+
+    def fetch_featured_published_projects
+      featured_published_projects = {}
+      PUBLISHED_PROJECT_TYPE_GROUPS.keys.each do |project_type|
+        featured_published_projects[project_type] = fetch_featured_projects_by_type(project_type)
+      end
+      return featured_published_projects
+    end
+
+    def project_and_featured_project_and_user_fields
+      [
+        :storage_apps__id___id,
+        :storage_apps__storage_id___storage_id,
+        :storage_apps__value___value,
+        :storage_apps__project_type___project_type,
+        :storage_apps__published_at___published_at,
+        :featured_projects__featured_at___featured_at,
+        :featured_projects__unfeatured_at___unfeatured_at,
+        :users__name___name,
+        :users__birthday___birthday,
+        :users__properties___properties,
+      ]
+    end
+
+    def fetch_featured_projects_by_type(project_type)
+      storage_apps = "#{CDO.pegasus_db_name}__storage_apps".to_sym
+      user_storage_ids = "#{CDO.pegasus_db_name}__user_storage_ids".to_sym
+      users = "dashboard_#{CDO.rack_env}__users".to_sym
```

Comment:
you should just be able to use `:users` here, because you are using DASHBOARD_DB to make the query.